test/core.jl

```diff
@@ -4862,3 +4862,12 @@ end
 @test_throws ErrorException isless(MyTime(1), now())
 
 end
+
+# issue #15240
+
+p15240 = ccall(:jl_realloc, Ptr{Void}, (Ptr{Void}, Csize_t), C_NULL, 10)
+ccall(:jl_free, Void, (Ptr{Void}, ), p15240)
```

Comment:
These should be written as tests, even if just testing that the returned value is `nothing`. Otherwise they don't get counted in the test stats at the end.